webapp/channels/src/components/widgets/inputs/users_emails_input.tsx

```diff
@@ -262,9 +262,22 @@ export default class UsersEmailsInput extends React.PureComponent<Props, State>
     MultiValueRemove = (props: MultiValueRemoveProps<EmailInvite | UserProfile, true>) => {
         const {children, innerProps} = props;
 
-        return (<div {...innerProps}>
-            {children || <CloseCircleSolidIcon/>}
-        </div>);
+        return (
+            <div
+                {...innerProps}
+                role='button'
+                tabIndex={0}
+                onKeyDown={(e) => {
+                    if (e.key === 'Enter' || e.key === ' ') {
```

Comment:
I was once stuck on which keys should trigger the click handler. Curious how did you decide that no other keys should be used here?

Reply:
I think we've just accepted that those two keys are the ones that are used to trigger buttons.